AABB_tree/doc/AABB_tree/Concepts/AABBRayIntersectionGeomTraits_3.h

```diff
@@ -46,21 +47,23 @@ class AABBRayIntersectionGeomTraits {
   typedef unspecified_type Cartesian_const_iterator_3;
```

Comment:
Just above this, it says "a model of `CartesianConstIterator3`"; it should be `CartesianConstIterator_3`.